css/custom.css

```diff
@@ -446,3 +446,16 @@ h3 {
 .content-wrapper {
     min-height: calc(100vh - 175px);
 }
+
+.script-container {
```

Comment:
These should be namespaced to gamerooom -- i.e. gameroom-script-container.

Reply:
Updated!